conf/config-default.yaml

```diff
@@ -64,8 +64,8 @@ apisix:
   # https_admin: true             # enable HTTPS when use a separate port for Admin API.
                                 # Admin API will use conf/apisix_admin_api.crt and conf/apisix_admin_api.key as certificate.
   admin_api_mtls:               # Depends on `port_admin` and `https_admin`.
-    admin_ssl_cert: ""             # Path of your self-signed server side cert.
-    admin_ssl_cert_key: ""         # Path of your self-signed server side key.
+    admin_ssl_cert: "cert/mtls_PLACE_HOLDER.crt"             # Path of your self-signed server side cert.
```

Comment:
The certificate for admin can't be a placeholder.

Reply:
already removed. thx!